ci/Jenkinsfile.windows

```diff
@@ -58,20 +60,24 @@ pipeline {
     stage('Package') {
       steps { script {
         windows.bundle(env.STATUS_CLIENT_EXE)
+        windows.bundle(env.STATUS_CLIENT_7Z)
```

Comment:
This way you'll run make twice and cause two rebuilds.

This is not pretty, but lets do this for now:
```
windows.bundle("${env.STATUS_CLIENT_EXE} ${env.STATUS_CLIENT_7Z}")
```
Which should allow it to run just one build. But we'll need to refactor this later.

Reply:
Fixed!